dev-libs/libtpms/libtpms-0.8.4.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Library providing software emultion of a TPM"
+HOMEPAGE="https://github.com/stefanberger/libtpms"
+SRC_URI="https://github.com/stefanberger/libtpms/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc"
+
+DEPEND="dev-libs/openssl:="
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-0.8.0-Remove-WError.patch"
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--disable-static \
+		--with-openssl \
+		--with-tpm2
+}
+
+src_install() {
+	default
+	find "${D}" -name '*.la' -delete || die
```

Comment:
trivial given both works but normally use `${ED}` here like the [policy-guide example](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0303).

good to check if it's still needed now and then too, perhaps .la files aren't installed anymore (haven't checked myself)

Reply:
Yes, it still install .la, but I have updated it to look like the example